unified-codes/apps/data-service/src/app/data.js

```diff
@@ -3,8 +3,7 @@ import { RESTDataSource } from 'apollo-datasource-rest';
 export class DgraphDataSource extends RESTDataSource {
   constructor() {
     super();
-    // TODO: lift this value out
-    this.baseURL = 'http://localhost:8080';
+    this.baseURL = process.env.NX_DGRAPH_ENDPOINT;
```

Comment:
could you do the same for the rx nav endpoint too please? add to `.env` and update here

Reply:
Done with a9661438f6c6f018f3199389168d1e1c2c8a11d0